packages/application-components/src/components/dialogs/internals/dialog-container.tsx

```diff
@@ -65,7 +57,15 @@ const GridArea = styled.div<GridAreaProps>`
   grid-area: ${(props) => props.name};
 `;
 
-const DialogContainer = (props: Props) => {
+const DialogContainer = ({
+  // TODO: t-shirt sizes are deprecated but we need to keep using them for
+  // backwards compatibility and to help with styling migration
+  // After the migration is done, we should change this default value to 13.
+  // t-shirt sizes then can be removed in a next breaking change release
+  size = 'l',
```

Comment:
Can't we set the default value already to `13`?

Reply:
Updated here: ace834344234d78743ed3fe4a71acc48bd67f117